src/test/scala/com/linkedin/feathr/offline/FeatureGenIntegTest.scala

```diff
@@ -278,6 +278,149 @@ class FeatureGenIntegTest extends FeathrIntegTest {
+        |
+        |            trips_today: {
+        |
+        |
+        |             def.sqlExpr: "trips_today + 1" //if_else(true, trips_today, 1)"
+        |
+        |
+        |
+        |            type: {
+        |                type: TENSOR
+        |                tensorCategory: DENSE
+        |                dimensionType: []
+        |                valType: INT
+        |            }
+        |
+        |            }
+        |
+        |            f_modified_streaming_count: {
+        |
+        |
+        |             def.sqlExpr: "trips_today + 1" //if_else(true, trips_today, 1)"
+        |
+        |
+        |
+        |            type: {
+        |                type: TENSOR
+        |                tensorCategory: DENSE
+        |                dimensionType: []
+        |                valType: INT
+        |            }
+        |
+        |            }
+        |
+        |
+        |                }
+        |            }
+        |
+        |            kafkaAnchor2: {
+        |                source: kafkaStreamingSource
+        |                key.sqlExpr: [driver_id]
+        |                features: {
+        |
+        |
+        |            trips_today2: {
+        |
+        |
+        |             def.sqlExpr: "trips_today + 1" //if_else(true, trips_today, 1)"
+        |
+        |
+        |
+        |            type: {
+        |                type: TENSOR
+        |                tensorCategory: DENSE
+        |                dimensionType: []
+        |                valType: INT
+        |            }
+        |
+        |            }
+        |
+        |            f_modified_streaming_count2: {
+        |
+        |
+        |             def.sqlExpr: "trips_today + 1" //if_else(true, trips_today, 1)"
+        |
+        |
+        |
+        |            type: {
+        |                type: TENSOR
+        |                tensorCategory: DENSE
+        |                dimensionType: []
+        |                valType: INT
+        |            }
+        |
+        |            }
+        |
+        |
+        |                }
+        |            }
+        |
+        |
+        |
+        |}
+        |
+        |sources: {
+        |
+        |
+        |
+        |kafkaStreamingSource: {
+        |    type: KAFKA
+        |    config: {
+        |        brokers: ["feathrazureci.servicebus.windows.net:9093"]
+        |        topics: [feathrcieventhub]
+        |
+        |        schema: {
+        |            type = "avro"
+        |            avroJson:"\n    {\n        \"type\": \"record\",\n        \"name\": \"DriverTrips\",\n        \"fields\": [\n            {\"name\": \"driver_id\", \"type\": \"long\"},\n            {\"name\": \"trips_today\", \"type\": \"int\"},\n            {\n                \"name\": \"datetime\",\n                \"type\": {\"type\": \"long\", \"logicalType\": \"timestamp-micros\"}\n            }\n        ]\n    }\n    "
```

Comment:
pretty print the schema so it's easer to read?

Reply:
HOCON file does not support it